diff --git a/lib/routes/jin10/category.ts b/lib/routes/jin10/category.ts
--- a/lib/routes/jin10/category.ts
+++ b/lib/routes/jin10/category.ts
@@ -170,7 +170,7 @@ export const route: Route = {
 async function handler(ctx) {
     const id = ctx.req.param('id');
     const data = await cache.tryGet(
-        'jin10:aa:${category}',
+        `jin10:aa:${id}`,
         async () => {
             const { data: response } = await got('https://4a735ea38f8146198dc205d2e2d1bd28.z3c.jin10.com/flash', {
                 headers: {
